reference/5.1/Microsoft.PowerShell.Core/About/about_History.md

```diff
@@ -41,6 +41,9 @@ command history.
   <kbd>F8</kbd>. Press <kbd>F8</kbd> again the next instance.
 - <kbd>F9</kbd> - Find a command by history ID. Type the history ID then press
   <kbd>F9</kbd>. Press <kbd>F7</kbd> to find the ID.
+- <kbd>#</kbd>`<string>`</kbd><kbd>Tab</kbd> - search the history for
+  `*<string>*` and returns the most recent match. Press <kbd>Tab</kbd> again to
```

Comment:
Should this be changed to "tab again to loop through previous matches." since it doesn't just find the previous match, but will loop?

Reply:
Added